src/interpreter_funcs/create_box.rs

```diff
@@ -95,6 +103,12 @@ impl Func for FuncCreateBox {
             Vector3::from(scale),
         );
 
+        if analyze {
+            analytics::report_mesh_analysis(&value)
+                .iter()
+                .for_each(|line| log(line.clone()));
```

Comment:
Shouldn't need to clone. Can move. Try `into_iter()`. Same goes for all the other reports.

Reply:
obsolete